### PR TITLE
PR to update to new name Heartcore

### PR DESCRIPTION
**Pull request by sofietoft**

### CODE REVIEW

File: Umbraco-Heartcore/Client-Libraries/index.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+versionFrom: 8.0.0
+---
+
+# Client Libraries
+
+In order for you to get the best start with Umbraco Heartcore and get an overview of the potential of the product, there are some SDK's (Software Development Kits) that you can try out. 
+
+## [Umbraco.Heartcore.Client.Net](https://github.com/umbraco/Umbraco.Headless.Client.Net)
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Client-Libraries/index.md]: Don't think we should rename this yet

File: Umbraco-Heartcore/Client-Libraries/index.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+.NET Core client library for the Umbraco Heartcore APIs. The library is based on Netstandard2.0 to support application development including Xamarin/UWP applications.
+
+This SDK includes 2 samples:
+
+* An [ASP.NET Core MVC sample](https://github.com/umbraco/Umbraco.Headless.Client.Net/tree/master/samples/Umbraco.Headless.Client.Samples.Web) site, with custom routing and controller hijacking.
+* A [console application](https://github.com/umbraco/Umbraco.Headless.Client.Net/tree/master/samples/Umbraco.Headless.Client.Samples.Console), with options using the Content Delivery and the Content Management API.
+
+## [Umbraco.Heartcore.Client.NodeJs](https://github.com/umbraco/Umbraco.Headless.Client.NodeJs)
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Client-Libraries/index.md]: Don't think we should rename this yet

File: Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md [hunk 3]
```diff
@@ -2,23 +2,23 @@
...
+When you log in to the Umbraco Backoffice, the first thing you will see if the Content section. This is where you will be creating content for your Umbraco Heartcore project. However, it will not be possible to create any content yet, as we will first need to define the content we are going to be creating.
 
 Content in Umbraco is based on **Document Types** which will define what type of data we can put into our content. A Document Type consists of a set of **Properties** - also called fields. A Property is made up of a **Data Type** which is a custom configuration of a **Property Editor**. Umbraco comes with a set of Property Editors, including a *Text Area*, a *Date Picker*, a picker for media items and many more.
 
-As we go through this tutorial and start builing the content for our Headless project, you will learn more about each of these concepts and how they all work together.
+As we go through this tutorial and start builing the content for our Heartcore project, you will learn more about each of these concepts and how they all work together.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md]:
```suggestion
As we go through this tutorial and start builing the content for our Umbraco Heartcore project, you will learn more about each of these concepts and how they all work together.
```

File: Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md [hunk 4]
```diff
@@ -139,7 +139,7 @@ Example:
 In the Content section, you will with this setup be able to create a News Area content node, and then create your News Articles under that node. 
 :::
 
-For our Headless project, we will want it to be possible to create blog posts under a blog area.
+For our Heartcore project, we will want it to be possible to create blog posts under a blog area.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md]:
```suggestion
For our Umbraco Heartcore project, we will want it to be possible to create blog posts under a blog area.
```

File: Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md [hunk 5]
```diff
@@ -188,7 +188,7 @@ However, once you've checked this permission setting on one Document Type, it wi
     * Doing this, will allow us to create content of the Blog Area type under our Home Page.
 5. **Save** the Document Type.
 
-With the permissions set, we are now ready to start working on the content for the Headless project.
+With the permissions set, we are now ready to start working on the content for the Heartcore project.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md]:
```suggestion
With the permissions set, we are now ready to start working on the content for the Umbraco Heartcore project.
```

File: Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md [hunk 6]
```diff
@@ -223,9 +223,9 @@ Select the **Media** section in the top navigation bar, and you can browse all t
 
 So far, our Content section looks like this:
 
-![Welcome to Headless Content node](images/Welcome-to-Headless-Content.png)
+![Welcome to Heartcore Content node](images/Welcome-to-Heartcore-Content.png)
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/Creating-a-Heartcore-project/index.md]:
```suggestion
![Welcome to Umbraco Heartcore Content node](images/Welcome-to-Heartcore-Content.png)
```

File: Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md [hunk 7]
```diff
@@ -36,9 +36,9 @@ Every item you have is considered a node. It could be a media item or content in
 
 ## Dashboard
 
-Every section in the Umbraco backoffice has a default dashboard associated with them. The first thing you will see when accessing the backoffice on your Umbraco Headless project is the "Welcome" dashboard, where you can find links to news and resources useful to your project. 
+Every section in the Umbraco backoffice has a default dashboard associated with them. The first thing you will see when accessing the backoffice on your Umbraco Heartcore project is the "Welcome" dashboard, where you can find links to news and resources useful to your project. 
 
-The Dashboard is where you can see what environment you are currently working on and you will also be able to navigate to the front-end of your headless project as well as navigate to the project on Umbraco.io.
+The Dashboard is where you can see what environment you are currently working on and you will also be able to navigate to the front-end of your Heartcore project as well as navigate to the project on Umbraco.io.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md]:
```suggestion
The Dashboard is where you can see what environment you are currently working on and you will also be able to navigate to the front-end of your Umbraco Heartcore project as well as navigate to the project on Umbraco.io.
```

File: Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md [hunk 8]
```diff
@@ -104,9 +104,9 @@ The log viewer is a view where you can go through all your logs. You are able to
 
 Here you will be able to manage your language variants. Depending on what plan you have chosen you will be able to have more that one language.
 
-### Headless
+### Heartcore
```
Comment (rasmusjp) [inline Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md]: Not sure about renaming this section (and tree node yet)

File: Umbraco-Heartcore/index.md [hunk 9]
```diff
@@ -0,0 +1,37 @@
...
+
+Reference documentation for the two APIs available, as well as details about common HTTP headers, versioning, REST Standard and how to work with authentication and authorization.
+
+- [Authentication and Authorization](API-Documentation/#authentication-and-authorization)
+- [Content Delivery API](API-Documentation/Content-Delivery)
+- [Content Management API](API-Documentation/Content-Management)
+
+## [Getting Started with Umbraco Heartcore](Getting-Started-Cloud/)
+
+In order to get started with Umbraco Heartcore you will need to create a new project via the Umbraco Cloud Portal and familiarize yourself with the features available. This includes the Heartcore section in the backoffice and creating API Keys for authentication and authorization purposes.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/index.md]:
```suggestion
In order to get started with Umbraco Heartcore you will need to create a new project via the Umbraco Cloud Portal and familiarize yourself with the features available. This includes the Umbraco Heartcore section in the backoffice and creating API Keys for authentication and authorization purposes.
```

File: Umbraco-Heartcore/API-Documentation/index.md [hunk 10]
```diff
@@ -4,25 +4,25 @@ versionFrom: 8.0.0
...
+The properties in the REST API, which starts with an underscore, are **system level properties**. That means that they are standard Umbraco properties, which cannot be changed via the API. This includes properties like `_id`, `_url`, `_createDate`, `_updateDate`, `_creatorName`, `_writerName`, `_level` and `_hasChildren`. These are all defined by Umbraco when Content is created or updated.
 
-The properties `_links` and `_embedded` are both part of the HAL specification and are implemented in the Umbraco Headless API accordingly.
+The properties `_links` and `_embedded` are both part of the HAL specification and are implemented in the REST API accordingly.
 
 ### API Browser
 
-In the Settings section in the Umbraco Backoffice, there is a Headless tree. From there you can use the API Browser to interact with both the Content Delivery and Content Management APIs. 
+In the Settings section in the Umbraco Backoffice, there is a Heartcore specific tree. From there you can use the API Browser to interact with both the Content Delivery and Content Management APIs.
```
Comment (rasmusjp) [inline Umbraco-Heartcore/API-Documentation/index.md]: Not sure about renaming the tree node

File: Umbraco-Heartcore/index.md (hunk 9 above)
Comment (sofietoft) [reply Umbraco-Heartcore/index.md]: Ah, this one is actually the section in the backoffice currently named 'Headless' - so, should we hold off renaming that for now, like you've suggested other places?

File: Umbraco-Heartcore/API-Documentation/index.md (hunk 10 above)
Comment (sofietoft) [reply Umbraco-Heartcore/API-Documentation/index.md]: Let's leave it as "Headless" for now then.

File: Umbraco-Heartcore/Client-Libraries/index.md (hunk 1 above)
Comment (sofietoft) [reply Umbraco-Heartcore/Client-Libraries/index.md]: I'll change these two back! 💪

File: Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md (hunk 8 above)
Comment (sofietoft) [inline Umbraco-Heartcore/Getting-Started-Cloud/The-Umbraco-Backoffice/index.md]:
```suggestion
### Headless
```

File: Umbraco-Heartcore/index.md (hunk 9 above)
Comment (rasmusjp) [reply Umbraco-Heartcore/index.md]: Ahhh, yes. Let's leave it as Headless for now, I've updated the suggestion

**Pull request merged**